### PR TITLE
Add container subcommands

### PR DESCRIPTION
**Pull request by baude**
where applicable, add the container commands to the container sub menu

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: cmd/podmanV2/containers/unpause.go [hunk 1]
```diff
@@ -27,6 +27,10 @@ var (
 )
 
 func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: unpauseCommand,
+	})
 	registry.Commands = append(registry.Commands, registry.CliCommand{
 		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
 		Command: unpauseCommand,
```
Comment (TomSweeneyRedHat) [inline cmd/podmanV2/containers/unpause.go:36]: nit, it would good to switch the order of lines 36 and 37 to match the other commands.

**Comment by TomSweeneyRedHat** [comment]
One nit for consideration, otherwise LGTM

**Comment by rhatdan** [comment]
@jwhonce and @mheon Have already discussed this. I forget the reason.

**Comment by jwhonce** [comment]
@baude I prefer separate cobra.Commands where the duplicated fields are copied, and unique fields like `Example` can be more correct.  I like to have the "fully qualified" command be the Parent and alias commands be the child. For example, `podman kill` pulls from `podman container kill`. They can live in the same containers.go file.

When the command is a different name, I have been creating new files.  For example, `images/images.go` and `images/list.go`.  Makes finding things much easier.

**Pull request closed without merging**